pkg/plugins/backendplugin/grpcplugin/client_v2.go

```diff
@@ -44,6 +45,11 @@ func newClientV2(descriptor PluginDescriptor, logger log.Logger, rpcClient plugi
 		return nil, err
 	}
 
+	rawSettings, err := rpcClient.Dispense("instanceSettings")
```

Comment:
@marefr @wbrowne -- what is the string key based on?  something in the SDK?

Reply:
https://github.com/grafana/grafana-plugin-sdk-go/blob/757e917dee73e80850b7ed5694127402444b39b5/backend/grpcplugin/serve.go#L26-L48